dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -375,7 +377,9 @@ class ApplicationsControllerTest < ::ActionController::TestCase
 
     test 'update sends a decision email once if status changed to decision and if associated with an RP' do
       sign_in @program_manager
-      Pd::Application::TeacherApplicationMailer.expects(:accepted).once
+      Pd::Application::TeacherApplicationMailer.expects(:accepted).
+        with(instance_of(TEACHER_APPLICATION_CLASS)).
+        returns(mock {|mail| mail.expects(:deliver_now)})
```

Comment:
I double-checked that this new expectation fails without the non-test code change.

Reply:
Perfect, a true assertion! Thank you